backend/user-service/pom.xml

```diff
@@ -46,20 +47,50 @@
             <artifactId>spring-boot-starter-test</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.mapstruct</groupId>
+            <artifactId>mapstruct</artifactId>
+            <version>${org.mapstruct.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.springdoc</groupId>
+            <artifactId>springdoc-openapi-starter-webmvc-ui</artifactId>
+            <version>2.0.0</version>
+        </dependency>
     </dependencies>
 
     <build>
         <plugins>
             <plugin>
-                <groupId>org.springframework.boot</groupId>
-                <artifactId>spring-boot-maven-plugin</artifactId>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.8.1</version>
                 <configuration>
-                    <excludes>
-                        <exclude>
```

Comment:
Nie wiem czemu usunąłeś exclude lombok - jest do generowane by default przez spring initializer

https://stackoverflow.com/questions/45202639/should-lombok-have-a-provided-scope-in-spring-boot-dependencies
https://stackoverflow.com/questions/74718709/why-i-have-to-add-exclusion-in-spring-boot-maven-plugin-while-using-lombok